src/js/components/partner_leads/Index.js

```diff
@@ -1,7 +1,11 @@
 import React, {PropTypes} from 'react';
+import toastr from 'toastr';
 
 class PartnerLeads extends React.Component {
   render() {
+    toastr.options.positionClass = "toast-top-right";
+    toastr.options.preventDuplicates = true;
```

Comment:
Would these toastr options still work if set in `componentWillMount` ? Setting these in the render method reset it as long as re-rendering occurs. So it doesn't feel quite right.